x-pack/plugins/secops/public/pages/hosts/translations.ts

```diff
@@ -14,6 +14,10 @@ export const NO_AUDITBEAT_INDICES = i18n.translate('xpack.secops.hosts.noAuditBe
   defaultMessage: "Looks like you don't have any auditbeat indices.",
 });
 
+export const KQL_PACE_HOLDER = i18n.translate('xpack.secops.hosts.kqlPlaceHolder', {
```

Comment:
Nice! Looks like you're following the good docs from i18n.

One slight thing, is that you probably meant this to be spelled `KQL_PLACE_HOLDER`

Reply:
lol